minvws/nl-kat-coordination
# Fix cron for last day of the month

**Note:** This PR is based on PR https://github.com/minvws/nl-kat-coordination/pull/3809/

### Changes
Small fix when the start day is the 28th, 29th, 30th or 31st of the month. Will be processed as the last day of the month ("L").

### Issue link
Closes #3810

### Demo

_Please add some proof in the form of screenshots or screen recordings to show (off) new functionality, if there are interesting new features for end-users._

### QA notes
- Create a scheduled report
- Select a date on the 28th (or 29th, 30th or 31st)
- Select recurrence "Monthly"
- Check if the cron is something like `h m L * *` where h is the hour, m the minute and L should be "L"

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
